metaflow/plugins/kfp/kfp.py

```diff
@@ -931,7 +912,7 @@ def _create_metaflow_step_op(
         # and still be a valid JSON string when loaded by the Python module.
         metaflow_execution_cmd: str = (
             " && python -m metaflow.plugins.kfp.kfp_metaflow_step"
-            f' --clean_volume_cmd "{step_variables.clean_volume_cmd}"'
+            f' --volume_dir "{step_variables.volume_dir}"'
```

Comment:
When `step_variables.volume_dir` is `None`, does it render to `None` and become `None` correctly in `kfp_metaflow_step`?

Reply:
My bad, you're right, it's resolve to `"None"` (a string). I now pass in an empty string and resolve that to `None` within `kfp_metaflow_step.py`.